packages/roosterjs-editor-api/lib/format/replaceWithNode.ts

```diff
@@ -56,9 +56,9 @@ export default function replaceWithNode(
     }
 
     if (range) {
-        let backupRange = editor.getSelectionRange();
+        const backupRange = editor.getSelectionRange();

```

Comment:
Is there a way we can/should move this logic into the core API instead of having it live here?

Reply:
If you want, you can call core.api.getSelectionRange() inside a core API.

Reply:
The problem is the following "exact match" computation for determining if it's an exact match even if unspecified. Should that be in the core api?

Reply:
Hmm, that's a good question. In that case I'm ok to leave it here.